cbsettings/importers.py

```diff
@@ -34,7 +34,7 @@ def load_module(self, name):
 
         # Unroll the settings into a new module.
         module = new_module(self.module_name)
-        for k, v in self.settings.items():
+        for k, v in list(self.settings.items()):
```

Comment:
Is there any reason why we would convert the generator in Python 3 to a list here?